### ISSUE TITLE
Mise à jour du slug sur changement de code

### ISSUE THREAD

**Issue by GwilhermPoullennec** [inventaire-des-orgues/portail#470]

**Comment by patou** [comment]
Il faut faire attention avec ça, c'est que si les sites d'associations d'orgues commencent à mettre des liens vers leurs orgues, si le slug change les liens ne fonctionneront plus...

**Comment by GwilhermPoullennec** [comment]
Exact. Ceci-dit le principal cas concerné est un déplacement d’orgue, auquel cas de toute façon le lien ne sera plus à jour : on peut le rompre.

Un autre cas plus problématique est le renommage des édifices (ex : nom d’une église qui n’en avait pas). Pour cette raison il serait intéressant d’avoir le plus rapidement possible les noms d’édifices.

Même problème avec les QRcodes.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by patou** [comment]
Ou alors il faut garder dans la base l'ancien slug. Et si lorsque l'on cherche un slug, on le trouve pas, on relance la recherche sur les anciens slugs en faisant une redirection vers le nouveau.
